tdrs-frontend/src/actions/auth.js

```diff
@@ -35,20 +35,34 @@ export const CLEAR_AUTH = 'CLEAR_AUTH'
  * the authentication data in the Redux store is cleared, and the user
  * is considered 'logged out', and can no longer access private routes.
  */
+
 export const fetchAuth = () => async (dispatch) => {
   dispatch({ type: FETCH_AUTH })
   try {
     const URL = `${process.env.REACT_APP_BACKEND_URL}/auth_check`
+    const instance = utils.axiosInstance
     const {
-      data: { user },
-    } = await axios.get(URL, {
+      data: { user, csrf },
+    } = await instance.get(URL, {
       withCredentials: true,
     })
+
+    instance.defaults.headers['X-CSRFToken'] = csrf
```

Comment:
Due to an issue in production that was not present in our development environments, the csrf token was not available in the cookies like we were expecting them to be. We decided to deal with this by passing csrf with the auth request, since we expected it to be exposed to the client via a cookie any way, we believe this solution to be as secure as using a cookie.